### ISSUE TITLE
AddExtendsAnnotationToModelFactoriesRector rule are not applying any diff

### ISSUE THREAD

**Issue by hmazter** [driftingly/rector-laravel#137]
I'm trying to use the `AddExtendsAnnotationToModelFactoriesRector` to add the `@extends` annotation to factory classes, but nothing is applied. Am I missing something?

### Factory class `database/factories/CategoryFactory.php`
```php
<?php

namespace Database\Factories;

use Illuminate\Database\Eloquent\Factories\Factory;

class CategoryFactory extends Factory
{
    protected $model = \App\Models\Category::class;

    public function definition()
    {
        return [
            'name' => $this->faker->sentence(),
        ];
    }
}
```

### Rector config `rector.php`
```php
<?php

declare(strict_types=1);

use Rector\Config\RectorConfig;

return static function (RectorConfig $rectorConfig): void {
    $rectorConfig->paths([
        __DIR__ . '/database',
    ]);

    // register a single rule
    $rectorConfig->rule(RectorLaravel\Rector\Class_\AddExtendsAnnotationToModelFactoriesRector::class);
};
```

**Comment by GeniJaho** [comment]
You might need a backslash before the rule name. Maybe.

**Comment by hmazter** [comment]
> You might need a backslash before the rule name. Maybe.

Thanks for the reply.
Unfortenly it makes no difference, and neither does importing the class.

**Comment by hmazter** [comment]
The same thing happens in a fresh Laravel install if the `@extends` is removed from the default UserFactory

**Comment by GeniJaho** [comment]
@hmazter In that case I think it's because of the latest Rector release https://github.com/rectorphp/rector/releases/tag/0.18.3, which breaks rules that modify docblocks.

There is a PR to support that release https://github.com/driftingly/rector-laravel/pull/135, but until it gets merged, you can use a fixed older version of core Rector like 0.18.2: `"rector/rector": "0.18.2",`.

**Comment by hmazter** [comment]
Thanks that worked! 👍
